### DISCUSSION TITLE
Is Spring endorsing the IntelliJ IDE?

### DISCUSSION THREAD

**Discussion by youcangetme** [awspring/spring-cloud-aws#134] [General]
I was submitting a request to upgrade the Spring Cloud AWS package and my TA came back questioning why the IDE config files were in this project. I had no logical explanation so I thought maybe Spring is now making IntelliJ their default IDE?

**Comment by maciejwalkowiak** [comment]
Intellij IDEA is a fantastic IDE we all in the team use, but there is nothing Intellij specific in Spring Cloud AWS. There should be no Intellij related files attached to your project after adding Spring Cloud AWS dependency.

Can you explain exactly what files were there?

**Reply by LucasJC** [reply]
Hi! I believe he was talking about the .idea folder at the root of the project

**Reply by youcangetme** [reply]
Gotta love developers who check in their IDE project files... In my shop that is akin to a Navy pilot landing on the wrong carrier.

**Reply by maciejwalkowiak** [reply]
.idea folder was added to the project like 5 years ago before I took it over and I just did not pay attention to it. Doesn't bring any value also doesn't harm, but since it creates noise, should be removed.
